### ISSUE TITLE
Assign days off to members instead whole group

### ISSUE THREAD

**Issue by tonesp** [microsoft/vsts-team-calendar#375]
Hi!
I created a group in my Active Directory, and I use this group to manage permissions in Azure DevOps.
All fine, but I cannot report Days off on the Calendar, because the dropdown shows the group name, instead the members.
There is any configuration that allows to show the members of the group? Or how can the members reports days off?

Thanks!

**Comment by rcomanne-rabobank** [comment]
Yes please, this is currently blocking our usage of the calendar, as everyone in our org is added by group...

**Comment by jasonbash01** [comment]
We are experiencing the same issue.   
RBAC Security groups are established to control the turnover of offshore contractors.  The problem with managing the RBAC Group limits that ability to enter individual days off from the Calendar - Only the Groups are listed


![Image](https://github.com/user-attachments/assets/887fe76c-06ad-4c9e-8c3b-c18619b182eb)
